core/ir/x86/decode.c

```diff
@@ -2591,7 +2591,6 @@ decode_common(dcontext_t *dcontext, byte *pc, byte *orig_pc, instr_t *instr)
                                         decode operands too */
```

Comment:
> i#6238: validate instructions: OP_push,OP_pop,OP_cmp,OP_pusha,OP_popa,OP_push_imm,OP_ret,OP_ret_far,OP_loopne,OP_loope,OP_loop,OP_movzx,OP_cmp,OP_rcr 

Title is very long and likely to be truncated in various summary tools.  How about shrinking a bit: "i#6238: Fill in categories for x86 push,pop,ret,loop,movzx,cmp,rcr".  (Leaving the full list in the body.)